server/schedule/labeler/rules.go

```diff
@@ -32,6 +32,8 @@ type LabelRule struct {
 const (
 	// KeyRange is the rule type that labels regions by key range.
 	KeyRange = "key-range"
+	// KeyRanges is the rule type that specifies a list of key ranges.
+	KeyRanges = "key-ranges"
```

Comment:
The name `key-ranges` is too similar to the `key-range`, what about using `key-range-list` instead?

Reply:
@rleungx suggests to keep only one type. So I'm going to merge the 2 types to one (name `key-range`, but is a list of ranges)